src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -275,14 +175,31 @@
            construct-base-url
            utils/validate-cli-base-url!)))))
 
+(defn export!
+  [outfile nodes-data]
+  (with-open [tar-writer (archive/tarball-writer outfile)]
+    (let [metadata (export-metadata (now))]
+      (log/info (:msg metadata))
+      (utils/add-tar-entry tar-writer metadata))
+    (doseq [tar-item nodes-data]
+      (log/info (:msg tar-item))
```

Comment:
I'm in favor of that, now that I watch it work. Maybe that, plus a log line indicating when it finishes?